### ISSUE TITLE
TouchUI Composite MultiField not working when accessed through action bar properties

### ISSUE THREAD

**Issue by NielsInc** [Adobe-Consulting-Services/acs-aem-commons#715]
When accessing page properties via the page (Open > Open properties), 
![image](https://cloud.githubusercontent.com/assets/7724067/15111045/d8e942c8-15e4-11e6-9630-c42d7361bb7d.png)
![image](https://cloud.githubusercontent.com/assets/7724067/15111069/fc3a2878-15e4-11e6-84f7-65037e8c5775.png)

a nested multifield is working: the saved values are loaded correctly and the nested multifield is displayed.
![multifield-on-page](https://cloud.githubusercontent.com/assets/7724067/15110933/1dbb009a-15e4-11e6-824a-b36d05868fe1.jpg)

When accessing the same page properties via the site console (View properties), 
![image](https://cloud.githubusercontent.com/assets/7724067/15111054/e80d0f14-15e4-11e6-8e6a-2f9040a78aa6.png)

the nested multifield is not working anymore: the values are not loaded and any nesting is not shown.
![multifield-on-site](https://cloud.githubusercontent.com/assets/7724067/15110932/1db8d126-15e4-11e6-948c-289c3dfa7b84.jpg)

We found out that the `dialog-ready` event is not fired in the site view dialog and even when it would be fired, the form submit listener would never be added to the  form, since the form does not have a `cq-dialog` class, which seems required.

**/apps/acs-commons/touchui-widgets/source/touchui-composite-multifield.js**
```javascript
  $document.ready(function () {
       addDataInFields();

       $document.on("dialog-ready", function() {
           var dialog = document.querySelector('form.cq-dialog');

           if(dialog){
               dialog.addEventListener('submit', collectDataFromFields, true);
           }
       });
   });
```

Could it be that some JavaScript libraries/events are not available in the site view, which are present in the page view?
If so, can this be fixed, or is this a limitation?

**Comment by schoudry** [comment]
@NielsInc yes it currently supports authoring only (`categories="[cq.authoring.dialog,acs-commons.cq.authoring.dialog]"`) ... but making it listen to events in other consoles (projects, sites, assets etc) would be a nice enhancement

**Comment by morly** [comment]
hi @all,
same issue here (not able to store new multifield values).
Any solution?
Greets

**Comment by justinedelson** [comment]
I've created a potential fix for this in #776. would appreciate anyone willing to test this out (on AEM 6.2). Here are SNAPSHOT packages: https://files.acrobat.com/a/preview/410e3504-1f33-4454-b341-21c85c05d774

**Comment by justinedelson** [comment]
this will be fixed in 2.6.8 and 3.2.10
